host/bcm283x/gpio.go

```diff
@@ -1191,6 +1251,10 @@ type driverGPIO struct {
 	gpioMemory *gpioMap
 	// gpioBaseAddr is needed for DMA transfers.
 	gpioBaseAddr uint32
+	// Whether uses the old pull resistor setup method before bcm2711.
```

Comment:
FTR documentation should start with the symbol being described, I can do it in a follow up.

Reply:
Fixed in a follow up.